README.md

```diff
@@ -27,14 +34,12 @@ open a ticket at https://bugs.php.net and reference the bug id using #NNNNNN.
     the magic_quotes behavior and therefore should not raise a warning at any
     time. The patch removes this warning
 
-We do not merge pull requests directly on github. All PRs will be
-pulled and pushed through http://git.php.net.
+We do not merge pull requests directly on GitHub. All PRs will be pulled and
+pushed through http://git.php.net.
```

Comment:
How about linking to `https://`?

Reply:
@kelunik added in the following squashed commit. :+1:  to [README.md] and [CONTRIBUTING.md] files.